src/docx-etl.py

```diff
@@ -4,58 +4,94 @@
 import subprocess
 import requests
 
-#from dotenv import load_dotenv
-#load_dotenv() #for local secret management with .env file
+# TODO: This code is identical to the code of my latest draft in src\rss-etl.py. Consider refactoring to avoid duplication.
+DEBUG = True  # Set to True to print debug messages
+if DEBUG:
+    from dotenv import load_dotenv
+    load_dotenv()  # Load environment variables from .env file
+sb_api_url = os.environ["SUPABASE_API_URL"]  # github actions secret management
+sb_api_key = os.environ["SUPABASE_API_KEY"]  # github actions secret management
 
-senate_rss_feed = "https://www.legis.state.pa.us/WU01/LI/RSS/SenateBills.xml"
-house_rss_feed = "https://www.legis.state.pa.us/WU01/LI/RSS/HouseBills.xml"
 
-sb_api_url = "https://vsumrxhpkzegrktbtcui.supabase.co"
-#sb_api_key = os.getenv("SUPABASE_API_KEY") #local secret management wtih .env file
-sb_api_key = os.environ["SUPABASE_API_KEY"] #github actions secret management 
-
-def dowload_bill_doc_file(bill_link):
-    bill_link = bill_link.replace("txtType=HTM", "txtType=DOC") #change link to doc download
+def download_bill_doc_file(bill_link: str) -> str:
+    """
+    Downloads a .doc file from a given link
+    :param bill_link: link to the .doc file
+    :return: path to the downloaded .doc file
+    """
+    bill_link = bill_link.replace("txtType=HTM", "txtType=DOC")  # change link to doc download
     response = requests.get(bill_link, stream=True)
     output_file_name = "bill.doc"
-    with open(output_file_name, "wb") as f: #add try catch here and return empty filname if it fails
+    with open(output_file_name, "wb") as f:  # add try catch here and return empty filename if it fails
```

Comment:
1. Shouldn't return the empty filename, should return `str` or `None`, i.e. `Optional[None]`
2. I recommend putting it in a temp folder instead of the main repo folder, like "temp/bill.doc", this way can can add "temp" to `.gitignore`

Reply:
Nice catch on both; addressing both in my upcoming commit.